rustfmt.toml

```diff
@@ -2,7 +2,7 @@ edition = "2021"
 array_width = 80
 chain_width = 80
 comment_width = 100
-wrap_comments = true
+wrap_comments = false
```

Comment:
I noted in the commit message (which I wish GitHub did a better job of surfacing in review) that this (nightly) feature is currently a bit broken in `rustfmt`. For now, we'll need to rely on editor configs for line wrapping, since `rustfmt` will mess up the formatting of doc comments in some scenarios, _removing_ newlines in some situations where it thinks one isn't needed for some reason. In our case specifically, it was merging a sublist into a parent list item for unknown reasons, and digging into it, it appears to be a known bug.

It's unfortunate though, because it was nice to be able to use it.